src/gui/sdlmain.cpp

```diff
@@ -2579,10 +2715,20 @@ void Config_Add_SDL() {
 	Pstring->Set_help("What resolution to use for fullscreen: 'original', 'desktop'\n"
 	                  "or a fixed size (e.g. 1024x768).");
 
-	pstring = sdl_sec->Add_string("windowresolution", on_start, "original");
-	pstring->Set_help("Scale the window to this size. Value 'original' will resize\n"
-	                  "window to the resolution picked by the emulated program.\n"
-	                  "Not supported when 'output' is set to 'surface'.");
+	pstring = sdl_sec->Add_string("windowresolution", on_start, "default");
+	pstring->Set_help(
+	        "Set window size to be used when running in windowed mode:\n"
+	        "  default:   Select the best option based on your\n"
+	        "             environment and other settings.\n"
+	        "  original:  Resize window to the resolution picked by\n"
+	        "             the emulated program.\n"
+#if SDL_VERSION_ATLEAST(2, 0, 5)
+		"  resizable: Make the emulator window resizable. (experimental\n"
+		"             option, works only with output=opengl)\n"
+#endif
+		"  <custom>:  Exact resolution to use, will scale the window\n"
+		"             content to this size, e.g. 1024x768.\n"
+	        "             Not supported when 'output' is set to 'surface'.");
```

Comment:
(Minor rewording of the description)

Scale the window content to the indicated dimensions, in WxH format.
For example: 1024x768.  Scaling is not performed for output=surface.

---
I like the term 'dimensions' over 'resolution' and 'size', because it specifically involves the measurement around the boundary of the window. Resolution, even in display terms, describes how small or "fine" something might be (ie: dot-pitch on a monitor or phone screen). 

Even though `size` is extremely generic, it's universally understood when discussion window or screens - so we could use that.  Suggest whatever we use, to apply that terms across all three descriptions here.